hack/Buchsbaum_frequency.py

```diff
@@ -0,0 +1,98 @@
+        Representation of the first particle species (e.g., 'p' for protons, 'D+'
+        for deuterium, or 'He-4 +1' for singly ionized helium-4). If no
+        charge state information is provided, then species-2 is assumed
+        to be singly charged.
+    Z1 : `float` or `~astropy.units.Quantity`, optional
+        The charge state for ion species #1. If not provided, it defaults to the charge number of ``particle1``.
+    Z2 : `float` or `~astropy.units.Quantity`, optional
+        The charge state for ion species #2. If not provided, it defaults to the charge number of ``particle2``.
+
+    Returns
+    -------
+    omega_BB : `~astropy.units.Quantity`
+        The Buchsbaum frequency of the plasma in units of radians per second.
+        Setting keyword ``to_hz=True`` will apply the factor of :math:`1/2π`
+        and yield a value in Hz.
+    Raises
+    ------
+    `TypeError`
+        If the magnetic field is not a `~astropy.units.Quantity` or
+        ``particle`` is not of an appropriate type.
+    `ValueError`
+        If the magnetic field contains invalid values or particle cannot
+        be used to identify a particle or isotope.
+    Warns
+    -----
+    : `~astropy.units.UnitsWarning`
+        If units are not provided, SI units are assumed.
+
+    Notes
+    -----
+    In a magnetized plasma, the presence of two ion species allows the perpendicular
+    component of the cold-plasma dielectric coefficient
+    math:: \epsilon_{\perp} to vanish at an angular frequency referred to as the
+    Buchsbaum frequency:cite:p:`buchsbaum:1960`., also called the bi-ion frequency or ion-ion hybrid frequency:cite:p:`vincena:2013`.
+    This frequency can be defined as:
+    .. math::
+    \omega_{BB} \equiv \sqrt{\frac{\omega_{p1}^{2}\omega_{c2}^{2} + \omega_{p2}^{2}\omega_{c1}^{2}}{\omega_{p2}^{2}+\omega_{p2}^{2}}}
+
+#   These are the references, but do they need to go into a global bibtex-type document?
+#   [1] S. J. Buchsbaum, Phys. Fluids 3, 418 (1960); Resonance in a Plasma with Two Ion Species; https://doi.org/10.1063/1.1706052
+#   [2] S. T. Vincena, W. A. Farmer, J. E. Maggs, and G. J. Morales, Phys. Plasmas, 20, 012111 (2013);
+#    Investigation of an ion-ion hybrid Alfvén wave resonator https://doi.org/10.1063/1.4775777
+    """
+
+    omega_c1 = gyrofrequency(B, particle1, signed=False, Z=Z1)
+    omega_c2 = gyrofrequency(B, particle2, signed=False, Z=Z2)
+    omega_p1 = plasma_frequency(n1, particle1, z_mean=Z1)
+    omega_p2 = plasma_frequency(n2, particle2, z_mean=Z2)
+
+    return np.sqrt((omega_p1**2 * omega_c2**2 + omega_p2**2 * omega_c1**2) / ( omega_p1**2 + omega_p2**2))
+
+omega_bb_ = buchsbaum_frequency
+"""Alias to `~plasmapy.formulary.frequencies.buchsbaum_frequency`."""
+
+omega_ii_ = buchsbaum_frequency
+"""Alias to `~plasmapy.formulary.frequencies.buchsbaum_frequency`."""
+
+omega_bi_ = buchsbaum_frequency
+"""Alias to `~plasmapy.formulary.frequencies.buchsbaum_frequency`."""
```

Comment:
```suggestion
o_bb_ = buchsbaum_frequency
"""Alias to `~plasmapy.formulary.frequencies.buchsbaum_frequency`."""

o_ii_ = buchsbaum_frequency
"""Alias to `~plasmapy.formulary.frequencies.buchsbaum_frequency`."""

o_bi_ = buchsbaum_frequency
"""Alias to `~plasmapy.formulary.frequencies.buchsbaum_frequency`."""
```

The miss-match naming to `__aliases__` is what's causing the doc build to currently fail.

Reply:
done. thanks.